src/components/ConnectWallet/Wallets/Alby.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <connection-details name="Alby" requires="lnd">
+    <step-list>
+      <step>
+        Add the Alby extension to your browser and open the app.
```

Comment:
Because you're utilizing the Tor connector, maybe add a note that the native companion is required?

Reply:
Good point, I have added this now.